frontend/src/theme/components/Avatar.ts

```diff
@@ -1,24 +1,61 @@
-import { ComponentMultiStyleConfig } from '@chakra-ui/react'
+import { avatarAnatomy as parts } from '@chakra-ui/anatomy'
+import { ThemingProps } from '@chakra-ui/react'
+import { PartsStyleFunction, SystemStyleObject } from '@chakra-ui/theme-tools'
 
-export const Avatar: ComponentMultiStyleConfig = {
-  parts: ['container', 'badge', 'usernameItem', 'usernameIcon'],
-  baseStyle: {
+import { textStyles } from '../textStyles'
+
+const baseStyle: PartsStyleFunction<typeof parts> = ({ colorScheme: c }) => {
+  return {
     container: {
-      backgroundColor: 'primary.500',
-      fontStyle: 'subhead-2',
-      width: '2.5rem',
-      height: '2.5rem',
+      bg: `${c}.500`,
       color: 'white',
+      textStyle: 'subhead-2',
     },
     badge: {
-      borderColor: 'white',
       bg: 'danger.500',
-      width: '0.55rem',
-      height: '0.55rem',
-      borderWidth: '0.0625rem',
-      position: 'absolute',
-      top: '2.125rem',
-      left: '2.125rem',
+      border: '1px solid white',
+      transform: 'none',
+    },
+  }
+}
+
+const getBadgePlacement = (size: ThemingProps['size']): SystemStyleObject => {
+  switch (size) {
+    // Update width and height calculation in the future if needed.
+    default:
+      return {
+        botton: '1px',
```

Comment:
```suggestion
        bottom: '1px',
```

/personalPreference: at xs size, it looks better with bottom 0 than bottom 1px

Reply:
NOOOOO I MERGED

Reply:
Sorry! It wasn't merged when I opened it to take a look 😅